client/landing/subscriptions/components/settings/site-settings/notify-me-of-new-posts-toggle.tsx

```diff
@@ -3,27 +3,35 @@ import { ToggleControl } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
 
 type NotifyMeOfNewPostsToggleProps = {
+	className?: string;
+	isDisabled?: boolean;
+	showJetpackAppHint?: boolean;
+	toggleId?: string;
 	value: boolean;
-	isUpdating: boolean;
 	onChange: ( value: boolean ) => void;
 };
 
 const NotifyMeOfNewPostsToggle = ( {
-	value = false,
-	isUpdating = false,
+	className = '',
+	isDisabled = false,
+	showJetpackAppHint = false,
+	value,
 	onChange,
 }: NotifyMeOfNewPostsToggleProps ) => {
 	const translate = useTranslate();
 
 	return (
-		<div className="setting-item setting-item__last">
+		<div className={ className }>
 			<ToggleControl
-				label={ translate( 'Receive web and mobile notifications' ) }
+				label={ translate( 'Notify me of new posts' ) }
 				onChange={ () => onChange( ! value ) }
 				checked={ value }
-				disabled={ isUpdating }
+				disabled={ isDisabled }
 			/>
-			{ value && (
+			<p className="notify-new-posts-toggle__hint">
+				{ translate( 'Receive web and mobile notifications for new posts from this site.' ) }
```

Comment:
Before we were showing this hint at one of the two places but now we will show it on all places as it helps clarify the toggle.